### ISSUE TITLE
Cannot compile swift example with Xcode 9.2

### ISSUE THREAD

**Issue by vladimirzivanov** [GoogleCloudPlatform/ios-docs-samples#71]
I cannot run swift example, I have got these errors after `pod install` command:
![screen shot 2017-12-21 at 10 14 55 am](https://user-images.githubusercontent.com/10206203/34248943-13ca78f6-e638-11e7-9d61-0f8e83c60e63.png)

**Comment by gerchicov-bp** [comment]
I have the same issue

**Comment by simformsolutions** [comment]
Any update on the same I am too facing the same issue.

**Comment by vladimirzivanov** [comment]
Yes, you need to do this: https://github.com/GoogleCloudPlatform/ios-docs-samples/blob/master/speech/Swift/Speech-gRPC-Streaming/BUILDFIXES
Change non modular headers.

**Comment by varun-naharia** [comment]
Just don't use normal pod install instead use ./INSTALL-COCOAPODS
